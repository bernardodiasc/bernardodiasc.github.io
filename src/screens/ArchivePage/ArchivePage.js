import React, { Component } from 'react'

import Meta from 'components/Meta'
import LayoutContainer from 'containers/LayoutContainer'

class ArchivePage extends Component {
  static defaultProps = {}

  render() {
    return (
      <LayoutContainer>
        <Meta
          title="Archive of content"
          description=""
        />
        to do:<br/>
        display a timeline/calendar of posts<br/>
        years, months and days are clickable<br/>
        each filter opens on its route<br/>
        and show available posts on PostsListing<br/>
      </LayoutContainer>
    )
  }
}

export default ArchivePage
