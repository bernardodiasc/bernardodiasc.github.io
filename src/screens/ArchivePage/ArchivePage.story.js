import React from 'react'
import { storiesOf } from '@storybook/react'
import { withKnobs } from '@storybook/addon-knobs/react'
import StoryRouter from 'storybook-router'
import Provider from 'stories/containers/Provider'
import Docs from 'stories/containers/Docs'

import ArchivePage from './ArchivePage'
import ArchivePageReadme from './README.md'
storiesOf('Screens/ArchivePage', module)
  .addDecorator(story => <Provider story={story()} />)
  .addDecorator(StoryRouter())
  .addDecorator(withKnobs)
  .add('Info', () => <Docs>{ArchivePageReadme}</Docs>)
  .add('Example', () => <ArchivePage />)
  .add('Minimal render', () => <ArchivePage />)
