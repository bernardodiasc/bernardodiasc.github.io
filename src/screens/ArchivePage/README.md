# ArchivePage

## How to use

```
import ArchivePage from 'screens/ArchivePage'
```

```
<ArchivePage />
```

## Properties

| propName | propType | defaultValue | isRequired |
| - | - | - | - |
