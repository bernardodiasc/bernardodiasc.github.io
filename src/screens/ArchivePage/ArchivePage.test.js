import React from 'react'
import { shallow } from 'enzyme'

import ArchivePage from './ArchivePage.js'

it('renders without crashing', () => {
  const component = shallow(<ArchivePage />)
  expect(component).toMatchSnapshot()
})
