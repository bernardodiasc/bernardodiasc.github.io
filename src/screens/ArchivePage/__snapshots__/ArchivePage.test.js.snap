// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Connect(LayoutContainer)>
  <Meta
    description=""
    image=""
    title="Archive of content"
  />
  <PageContent>
    <h1>
      To do:
    </h1>
    display a timeline/calendar of posts
    <br />
    years, months and days are clickable
    <br />
    each filter opens on its route
    <br />
    and show available posts on PostsListing
    <br />
  </PageContent>
</Connect(LayoutContainer)>
`;
